test/unit/aggregate_store_test.rb

```diff
@@ -18,22 +18,28 @@ def add(attr, message)
     end
   end
 
+  def elasticsearch_store
+    @elasticsearch_store ||=
+      begin
+        store = Class.new
+        store.send(:include, Aggregate::AggregateStore)
+        store.define_singleton_method(:aggregate_db_storage_type) { :elasticsearch }
+        store
+      end
```

Comment:
Could this just be a normal class? I'm not seeing what benefit we get from building a new class through methods.
```ruby
class ElasticsearchStore
  include Aggregate::AggregateStore

  def self. aggregate_db_storage_type
    :elasticsearch
  end
end
```

Reply:
We need to dynamically create classes so that changes made to each class do not persist between tests.